great-idea/css/services.css

```diff
@@ -0,0 +1,296 @@
+    margin: 0 auto;
+}
+
+.navigation {
+    display: flex;
+    justify-content: space-between;
+    font-size: 20px;
+    margin: 20px 0;
+}
+
+.navigation a {
+    color: gray;
+    text-decoration: none;
+}
+
+.services-header-img {
+    margin-bottom: 20px;
+}
+
+.services {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    border-bottom: 1px solid black;
+    padding-bottom: 20px;
+}
+
+.services h3 {
+    font-size: 24px;
+    font-weight: bold;
+}
+
+.blocks {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.digital-design {
+    background: whitesmoke;
+    display: flex;
+    flex-wrap: wrap;
+    width: 420px;
+    align-content: flex-start;
+    border: 1px solid black;
+    margin-top: 25px;
+}
+
+.digital-design h3 {
+    font-size: 20px;
+    padding: 15px;
+}
+
+.digital-design p {
+    align-content: flex-start;
+    padding: 0 15px;
+}
+
+button {
+    justify-content: flex-start;
+    text-align: center;
+    font-size: 16px;
+    border-radius: 5px;
+    margin: 15px;
+    width: 150px;
+    height: 30px;
+}
+
+.ux-ui {
+    background: whitesmoke;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    width: 420px;
+    align-content: flex-start;
+    border: 1px solid black;
+    margin-top: 25px;
+    margin-left: 40px;
+}
+
+.ux-ui h3 {
+    font-size: 20px;
+    padding: 15px;
+}
+
+.ux-ui p {
+    align-content: flex-start;
+    padding: 0 15px;
+}
+
+.digital-marketing {
+    background: whitesmoke;
+    display: flex;
+    flex-wrap: wrap;
+    width: 420px;
+    align-content: flex-start;
+    border: 1px solid black;
+    margin-top: 25px;
+}
+
+.digital-marketing h3 {
+    font-size: 20px;
+    padding: 15px;
+}
+
+.digital-marketing p {
+    align-content: flex-start;
+    padding: 0 15px;
+}
+
+.web-development {
+    background: whitesmoke;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    width: 420px;
+    align-content: flex-start;
+    border: 1px solid black;
+    margin-top: 25px;
+    margin-left: 40px;
+}
+
+.web-development h3 {
+    font-size: 20px;
+    padding: 15px;
+}
+
+.web-development p {
+    align-content: flex-start;
+    padding: 0 15px;
+}
+
+.ios-development {
+    background: whitesmoke;
+    display: flex;
+    flex-wrap: wrap;
+    width: 420px;
+    align-content: flex-start;
+    border: 1px solid black;
+    margin-top: 25px;
+}
+
+.ios-development h3 {
+    font-size: 20px;
+    padding: 15px;
+}
+
+.ios-development p {
+    align-content: flex-start;
+    padding: 0 15px;
+}
+
+.android-development {
+    background: whitesmoke;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    width: 420px;
+    align-content: flex-start;
+    border: 1px solid black;
+    margin-top: 25px;
+    margin-left: 40px;
+}
+
+.android-development h3 {
+    font-size: 20px;
+    padding: 15px;
+}
+
+.android-development p {
+    align-content: flex-start;
+    padding: 0 15px;
+}
```

Comment:
i was wondering why this was so long! notice that these all say pretty much the same thing? having a single class like "services-box" on each and every one of these can reduce the redundancy and you only have to apply all these styles once.